pvlib/irradiance.py

```diff
@@ -2267,6 +2267,136 @@ def erbs(ghi, zenith, datetime_or_doy, min_cos_zenith=0.065, max_zenith=87):
     return data
 
 
+def erbs_driesse(ghi, zenith, datetime_or_doy=None, dni_extra=None,
+                 min_cos_zenith=0.065, max_zenith=87):
+    r"""
+    Estimate DNI and DHI from GHI using the continuous Erbs-Driesse model.
+
+    The Erbs-Driesse model [1]_ is a new formulations of the original Erbs
```

Comment:
"a *new* formulation" may eventually feel not so new. "a reformulation" or something like that might be more enduring.

Reply:
"New" vs. "original" should be enduring, but I have nevertheless reformulated the string and made a new commit.